en/docs/test/cURL.md

```diff
@@ -1,3 +1,23 @@
 # cURL
+Choreo allows you to test your APIs using the integrated OpenAPI Console, the generated cURL command, or by integrating Postman to Choreo. Once you have created an API and deployed it, it is ready to be tested. 
 
-A how-to guide on how to use cURL to test the component. 
\ No newline at end of file
+You can follow the steps below to test your REST API using the Choreo-generated cURL command :
+
+# Test your REST API
+
+!!! note
+    If you are on the deployment tab, click **Test**. in the **Development** card and continue from step 4 onwards. 
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
+
+2. Select and click on the API component you want to test from the components list. 
```

Comment:
Suggestion: "In the component list, find the API component that you want to test and click it."

Reply:
done